org.speed.dreams.yaml

```diff
@@ -0,0 +1,166 @@
+app-id: org.speed.dreams
```

Comment:
Is this ID used in the project or somewhere? It’s rather strange. Looking at the URL something like `org.speed_dreams.SpeedDreams` would be more accurate.